mapie/multi_label_classification.py

```diff
@@ -122,7 +125,12 @@ class MapieMultiLabelClassifier(BaseEstimator, ClassifierMixin):
      [False  True False]
      [False  True False]]
     """
-    valid_methods_ = ["crc", "rcps"]
+    valid_methods_ = ["crc", "rcps", "ltt"]
+    valid_metric_ = ['precision', 'recall']
+    valid_methods_by_metric_ = {
+        "precision": ["ltt"],
+        "recall": ["rcps", "crc"]
+    }
```

Comment:
```suggestion
    valid_methods_by_metric_ = {
        "precision": ["ltt"],
        "recall": ["rcps", "crc"]
    }
    valid_metric_ = {metric for metric in valid_methods_by_metric_}
    valid_methods_ = {method for l in valid_methods_by_metric_ for method in l}
```

Reply:
Doesn't work, l is a string and when you parcour a string you return each letter that compose that string (valid_methods).

Reply:
Can you adapt this code in order to define methods and metrics only one time?

Reply:
Done